apps/studio/components/layouts/ProjectLayout/LayoutHeader/FeedbackDropdown/FeedbackDropdown.tsx

```diff
@@ -21,7 +22,12 @@ const FeedbackDropdown = () => {
           <span className="hidden md:flex">Feedback</span>
         </Button>
       </PopoverTrigger_Shadcn_>
-      <PopoverContent_Shadcn_ side="bottom" align="end" className="w-full p-0">
+      <PopoverContent_Shadcn_
```

Comment:
For some reason the popover closes as soon as you click send... perhaps this needs to be controlled?